modules/inflowwind/src/InflowWind_Subs.f90

```diff
@@ -1665,14 +1658,6 @@ END SUBROUTINE CalculateOutput
 !> this routine calculates a rotor-averaged mean velocity, DiskVel 
 SUBROUTINE InflowWind_GetSpatialAverage( Time, InputData, p, x, xd, z, OtherStates, m, MeanVelocity, ErrStat, ErrMsg )
```

Comment:
I would rename that function to GetRotorSpatialAverage. It threw me off, because the inputs and number of points are already hardcoded, so it's not a generic function to compute an average.

Reply:
I agree with renaming it. I'll get that in soon.